### ISSUE TITLE
[2.13.1] Retrying a transition doesn't change the URL

### ISSUE THREAD

**Issue by boris-petrov** [emberjs/ember.js#15251]
This was working fine in 2.13.0. We're doing something like in this Twiddle:

https://ember-twiddle.com/4527e05572935830785ef60c863564f9

In the Twiddle it works fine but in our production code it doesn't. Try going to the `other` route - it will then transfer you to the `login` controller. If you click the `Login` button it will then "log you in" and retry the last transition - and in our production code this leaves the URL to be `/login` instead of changing it to be `/other`.

Interesting thing is that this only happens in our production case if you directly hit the `other` route after a refresh when you're not logged in - i.e. you enter directly the URL with `other` and press Enter.

I'm not sure what else I could tell you, please ask if you need more information.

**Comment by cbroeren** [comment]
We have the same problem since release 2.13.1
Glad we're testing this, so our tests are failing now too. Hope it will be fixed soon.

**Comment by workmanw** [comment]
Seems like it could be related to this: https://github.com/emberjs/ember.js/issues/15190

**Comment by rwjblue** [comment]
Thank you @workmanw. Yes, I suspected that it was due to me updating the yarn.lock incorrectly 😩.

I'll dig in and confirm this morning and get a 2.13.2 out.

**Comment by rwjblue** [comment]
Confirmed in the `yarn.lock` diff from 2.13.0 to 2.13.1:

```diff
 router_js@^1.2.4:
-  version "1.2.7"
-  resolved "https://registry.yarnpkg.com/router_js/-/router_js-1.2.7.tgz#f5cd153d87e31941d3a2bbd81700d5884c7bfc44"
+  version "1.2.5"
+  resolved "https://registry.yarnpkg.com/router_js/-/router_js-1.2.5.tgz#ea211eddada3f3d92f5e6d9c24a5b57288752f75"
```

**Comment by rwjblue** [comment]
Fixed in v2.13.2 (should be published in ~ 10 min).
